specification/eventgrid/resource-manager/Microsoft.EventGrid/stable/2019-06-01/EventGrid.json

```diff
@@ -0,0 +1,3497 @@
+          },
+          "EventSubscriptions_GetForResource": {
+            "$ref": "./examples/EventSubscriptions_GetForResource.json"
+          },
+          "EventSubscriptions_GetForCustomTopic": {
+            "$ref": "./examples/EventSubscriptions_GetForCustomTopic.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "EventSubscriptions"
+        ],
+        "summary": "Create or update an event subscription.",
+        "description": "Asynchronously creates a new event subscription or updates an existing event subscription based on the specified scope.",
+        "operationId": "EventSubscriptions_CreateOrUpdate",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "scope",
+            "in": "path",
+            "description": "The identifier of the resource to which the event subscription needs to be created or updated. The scope can be a subscription, or a resource group, or a top level resource belonging to a resource provider namespace, or an EventGrid topic. For example, use '/subscriptions/{subscriptionId}/' for a subscription, '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}' for a resource group, and '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{resourceProviderNamespace}/{resourceType}/{resourceName}' for a resource, and '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.EventGrid/topics/{topicName}' for an EventGrid topic.",
+            "required": true,
+            "type": "string",
+            "x-ms-skip-url-encoding": true
+          },
+          {
+            "name": "eventSubscriptionName",
+            "in": "path",
+            "description": "Name of the event subscription. Event subscription names must be between 3 and 64 characters in length and should use alphanumeric letters only.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "eventSubscriptionInfo",
+            "in": "body",
+            "description": "Event subscription properties containing the destination and filter information.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/EventSubscription"
+            }
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "EventSubscription CreateOrUpdate request accepted.",
+            "schema": {
+              "$ref": "#/definitions/EventSubscription"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 400 Bad Request.\n\n * 500 Internal Server Error."
+          }
+        },
+        "x-ms-examples": {
```

Comment:
It looks like the earlier version had some more examples here, can you please double-check if they are still required?

Reply:
Thx. Fixed